### ISSUE TITLE
Feature request: More keyboard shortcuts

### ISSUE THREAD

**Issue by y-lee** [yuliskov/SmartYouTubeTV#606]
Version: 6.17.702 / web_20200726_00_RC00

In Pro Alt, the menu for selecting playback speed has changed. The additional increments (1.10X and 1.15X) causes problems with the macro system I use to adjust playback speed with one remote button press.

May I request the addition of more keyboard shortcuts? I have in mind four types:

* `<` and `>` to decrease and increase playback speed.
* `c` to toggle subtitles.
* `0` through `9` to jump to the *x*0% of a video.
* `1` through `9` to jump to the first through ninth entry on a menu.

The first three are already implemented in stock YouTube.

**Comment by onolox** [comment]
+1, more shortcuts are needed, especially playback speed.
A cool feature is to select our own shortcuts, something like a list, then select an item and open an modal an click the that key you want assingned to a function, but this is a dream.

, and . to change speeds are a real necessity!

**Comment by yuliskov** [comment]
Hi, guys. Are you using non-default controllers?

**Comment by y-lee** [comment]
> Hi, guys. Are you using non-default controllers?

Yes, I'm using a remote control that I can add functionality to. But even people using the remote control that came with their Shield TV or other Android TV box might have keyboards connected.

**Comment by onolox** [comment]
These controllers are known as "air mouse", you have a normal controller and a keyboard.
But this applies to a normal keyboard attached to android tv box.

**Comment by yuliskov** [comment]
Fabio, "air mouse" is an additional feature that makes the controller act
like a mouse. Number of buttons on such controllers may differ depending on
the model. I need to know the exact models of controllers you guys are
using.

**Comment by y-lee** [comment]
Using the same keyboard bindings as the web version of YouTube is fine. Let us worry about binding the keys to our own remote controls.

**Comment by y-lee** [comment]
To put another way, using keyboard keys (already supported in Android) is the easiest and most universal way of implementing remote control. Anyone can use them, whether using a keyboard, or something like Flirc (which can emulate keyboard).

**Comment by y-lee** [comment]
YouTube keyboard shortcuts: https://support.google.com/youtube/answer/7631406

**Comment by onolox** [comment]
Yes, binding normal keyboard shortcuts is the best solution.
 But I think is best to not use combos like "Shit+something". Using just a sigle key, is fine. Like "f" for fullscreen on off.

In my case I REALLY need keys to increase and decrease speed.

**Comment by yuliskov** [comment]
Yeechang Lee <notifications@github.com>, thx for the tip. I'll try to
achieve the same in my app. However, I can't say anything about a terms.

Regards, Yuriy
